source/software/r_package_management.rst

```diff
@@ -105,17 +88,18 @@ If the result is blank, or reports that conda can not be found, modify
 the \`PATH\` environment variable appropriately by adding miniconda's bin
 directory to PATH.
 
-Creating a new conda environment is straightforward::
+The next step is to create a new conda environment which can be done as follows::
+
+   $ conda create -n science -c conda-forge r-base=4.3.2 r-essentials
+   
 
-   $ conda create -n science -c r r-essentials r-rodbc
+This command creates a new conda environment called "science", and installs R 4.3.2
+from the conda-forge channel as well as the r-essentials bundle which includes number
+of commonly used R packages such as ggplot2, glmnet, dplyr, tidyr, and shiny.
 
-This command creates a new conda environment called science, and
-installs a number of R packages that you will probably want to have
-handy in any case to preprocess, visualize, or postprocess your data.
-You can of course install more, depending on your requirements and
-personal taste.
+.. note::
 
-A lot of bioconda and bioconductor packages are not in sync with their dependencies, therefore you may need to create a separate environment for each of those packages to avoid conflicts.
+   A lot of bioconda and bioconductor packages are not in sync with their dependencies, therefore you may need to create a separate environment for each of those packages to avoid conflicts.
```

Comment:
For bioconda I'm not sure, but with bioconductor this is because there are 2 releases per year, so dependencies can become somewhat older at some point. I would rather say here: start with installing the bioconductor (and bioconda?) packages and let conda figure out the dependencies.

Reply:
I did not change the content of this remark, I just made it a propper `.. note:: `. I am not sure what the reason was behind this remark. Do you know @moravveji?

Reply:
I don't know where this comment comes from, but it is a very relevant one. So, I'd keep it.